src/assets/styles/footer.scss

```diff
@@ -0,0 +1,88 @@
+.footer {
+  display: inline-block;
+  background: $light-gray;
+  height: 10em;
+  width: 100%;
+
+  .footer-buttons {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+
+  a.btn {
+    margin: 1.5em 0.6em 0.3em 0.6em;
+    color: white;
+    text-decoration: none;
+    flex-grow: 1;
+    font-size: 1em;
+    text-align: center;
+    position: relative;
+    min-width: calc(50% - 2.5em);
+  }
+
+  .privacy-policy {
+    margin: 0 1em;
+  }
+
+  .host-btn {
+    background: $aclu-blue;
+  }
+
+  .share-btn {
+    background: $facebook-blue;
+
```

Comment:
Just a thought -- the share button stuff might be easier to manage with this FB logo image inline, for example this would allow something with a bit simpler html and css:

![image](https://cloud.githubusercontent.com/assets/1724544/25467330/5537e256-2ac2-11e7-9b7e-d58419bfa495.png)

And I think then it wouldn't need different styles for mobile/desktop